spec/mail/mailbox.mdown

```diff
@@ -221,7 +232,7 @@ If the mailbox has `mayDelete == false`, the destruction MUST be rejected with a
 
 A mailbox MUST NOT be destroyed if it still has any child mailboxes. Attempts to do so MUST be rejected with a `mailboxHasChild` error.
 
-A mailbox MUST NOT be destroyed if it has any messages assigned to it. Attempts to do so MUST be rejected with a `mailboxHasMessage` error.
+Before destroying the mailbox, all messages currently assigned to it must be removed from the mailbox. If a message is not in any other mailboxes, it MUST be added to the mailbox given by the *onLastReferenceMoveTo* argument, if given, otherwise destroyed.
```

Comment:
```
Before destroying the mailbox, all messages currently assigned to it must be removed from the mailbox.
```

Should rather be:

```
Before destroying the mailbox, all messages currently assigned to it SHOULD be removed from the mailbox.
```

Reply:
No, it can't be a SHOULD; what else would the server do? It's an interoperability issue if it's not a must.